### PR TITLE
feat(cart): add migration doc

### PR DESCRIPTION
**Pull request by dpellier**

### CODE REVIEW

File: packages/storybook/stories/components/cart/migration.from.17.x.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Current component issues
+
+### Performance & Complexity
+
+The whole component was composed of 9 web-components, each with their own rendering lifecycle.
+
+Each of those were relying on DOM queries to update their states accordingly to their parents/siblings, which
+can become quite problematic performance-wise.
+
+Plus to use this component, integrators had to learn about the 9 different interfaces and all the possible combinations.
```
Comment (Leotheluck) [inline packages/storybook/stories/components/cart/migration.from.17.x.mdx:21]: Also, in order to use this component, the integrators had to learn about the 9 different interfaces and all their possible combinations.

File: packages/storybook/stories/components/cart/migration.from.17.x.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Internal logic
+
+ODS component tends to be presentational component, the internal logic should be handle by people integrating the component.
```
Comment (Leotheluck) [inline packages/storybook/stories/components/cart/migration.from.17.x.mdx:25]: ODS components tend to be presentational, thus the internal logic should be handled by the integrators.

File: packages/storybook/stories/components/cart/migration.from.17.x.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Current Cart implementation was trying to enforce some of the logic (ex: quantity computation).
```
Comment (Leotheluck) [inline packages/storybook/stories/components/cart/migration.from.17.x.mdx:27]: The current implementation of Cart was trying to enforce some of the logic (e.g. quantity computation).

File: packages/storybook/stories/components/cart/migration.from.17.x.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+This component is basically a very simple top-to-bottom layout, where each section can be rendered using `ods-text` or even
+a simple text element. You can also use `ods-divider` to add some separation if needed.
+
+All the colors and style properties can be found in the exposed design tokens.
+
+## OVH components?
+
+It is possible, in a near future, that very OVH oriented components like this one, find their way back into ODS,
+probably as a separate tag namespace.
```
Comment (Leotheluck) [inline packages/storybook/stories/components/cart/migration.from.17.x.mdx:39]: It is possible, in the near future, that OVHcloud-oriented components like this one find their way back into ODS, probably as a separate tag namespace.

File: packages/storybook/stories/components/cart/migration.from.17.x.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Current Cart implementation was trying to enforce some of the logic (ex: quantity computation).
+
+## How to use Cart in v18
+
+This component is basically a very simple top-to-bottom layout, where each section can be rendered using `ods-text` or even
+a simple text element. You can also use `ods-divider` to add some separation if needed.
+
+All the colors and style properties can be found in the exposed design tokens.
+
+## OVH components?
```
Comment (feoche) [inline packages/storybook/stories/components/cart/migration.from.17.x.mdx:36]: I'd prefer to exclude this paragraph, as we are not able to know at which level component can be re-included inside ODS. It's something we need to communicate for sure, but preferably with direct teams.

**Review by feoche — CHANGES_REQUESTED** [review]

**Review by aesteves60 — APPROVED** [review]

**Review by astagnol — APPROVED** [review]

**Pull request merged**